integration-tests/springmvc-tests/src/test/java/io/servicecomb/demo/springmvc/tests/SpringMvcIntegrationTestBase.java

```diff
@@ -162,6 +173,66 @@ public void ableToPostBytes() throws IOException {
     assertEquals(3, result.length);
   }
 
+  @Test
+  public void ableToUploadFile() throws IOException {
+    String file1Content = "hello world";
+    String file2Content = "bonjour";
+
+    MultiValueMap<String, Object> map = new LinkedMultiValueMap<>();
+    map.add("file1", new FileSystemResource(newFile(file1Content).getAbsolutePath()));
+    map.add("someFile", new FileSystemResource(newFile(file2Content).getAbsolutePath()));
```

Comment:
maybe, we lost SDK as consumer to send file testcase?

Reply:
Yes, It's better to add a test case in Demos to use ServiceComb cutomized RestTemplate

Reply:
sure, i am looking into it

Reply:
does vertx support sending multipart files?

it seems such a feature is supported now
http://vertx.io/docs/vertx-web-client/java/

Reply:
still did not write a SDK consumer to test upload?

Reply:
added in test case already